Software/dist_windows/script32.iss

```diff
@@ -57,18 +57,13 @@ Source: "content/Qt5Network.dll"; DestDir: "{app}"; Flags: ignoreversion
 Source: "content/Qt5SerialPort.dll"; DestDir: "{app}"; Flags: ignoreversion
 Source: "content/Qt5Widgets.dll"; DestDir: "{app}"; Flags: ignoreversion
 Source: "gpl-3.0.txt"; DestDir: "{app}"; Flags: ignoreversion
-Source: "content/icudt54.dll"; DestDir: "{app}"; Flags: ignoreversion
-Source: "content/icuin54.dll"; DestDir: "{app}"; Flags: ignoreversion
-Source: "content/icuuc54.dll"; DestDir: "{app}"; Flags: ignoreversion
```

Comment:
If it works out fine, it will save 50% of the installer file size. Which is also great.

No, I meant the black rectangles. But I seem to recall that these also showed up when I added a virtual device months ago. So I guess it's just small bug in the UI.